DBM-Naxx/FrostwyrmLair/Sapphiron.lua

```diff
@@ -65,14 +73,27 @@ mod.CHAT_MSG_RAID_BOSS_EMOTE = mod.CHAT_MSG_MONSTER_EMOTE -- used to be a normal
 
 function mod:OnSync(event)
 	if event == "DeepBreath" then
-		timerIceBlast:Show()
+		if self.Options.PlaySoundOnIceBlast then
+			PlaySoundFile("Sound\\Creature\\HeadlessHorseman\\Horseman_Beckon_01.wav")
+		end
+		if mod:IsDifficulty("heroic25") then
+			timerIceBlast:Start(7.8)
+		else
+			timerIceBlast:Show()
+		end
 		timerLanding:Update(14)
 		self:ScheduleMethod(14.5, "Landing")
 		warnDeepBreath:Show()
 	end
 end
 
 function mod:Landing()
+	if mod:IsDifficulty("heroic25") then
+		timerFrozenOrb:Start()
+		timerFrozenOrb:Schedule(30, 25)
+		warnFrozenOrb:Schedule(30)
```

Comment:
Can this be (30, 25)?

Reply:
It will in 30s create a timer that last 25 as intended

Reply:
Why i mean is it will create timer that last 30s, and then after 30s it will create another timer that last 25s,as was suggested by holy